### PR TITLE
Provide a cli command to dump idgraph hashes

### PR DESCRIPTION
**Pull request by Kailai-Wang**
### Context

as topic.

This PR adds a `dump-id-graph-hash` subcommand to the CLI with default shard.
It retrieves the IDGraph hash map via the `PublicGetter:: all_id_graph_hash`

### CODE REVIEW

**Review by BillyWooo — APPROVED** [review]
All looks good to me.

File: tee-worker/app-libs/stf/src/getter.rs [hunk 1]
```diff
@@ -99,6 +99,8 @@ pub enum PublicGetter {
 	some_value,
 	#[codec(index = 1)]
 	nonce(Identity),
+	#[codec(index = 2)]
+	all_id_graph_hash,
```
Comment (kziemianek) [inline tee-worker/app-libs/stf/src/getter.rs:103]: Can we have this getter in two variants ? 

`MY|ALL` ?

File: tee-worker/app-libs/stf/src/getter.rs [hunk 2]
```diff
@@ -278,6 +280,8 @@ impl ExecuteGetter for PublicGetter {
 				} else {
 					None
 				},
+			PublicGetter::all_id_graph_hash =>
+				Some(IdentityManagement::all_id_graph_hash().encode()),
```
Comment (kziemianek) [inline tee-worker/app-libs/stf/src/getter.rs:284]: May this cause problems if somebody starts to flood node with such getters ?

Comment (Kailai-Wang) [reply tee-worker/app-libs/stf/src/getter.rs:284]: Then it's not (only) a problem of this one: it applies to any RPC interface: getter, trusted getter or trusted calls. It also applies to any relay/parachains - you could spam costly requests like `state_getKeys`

**Review by kziemianek — APPROVED** [review]

**Pull request merged**